rfc/src/rfcs/0009-function-contracts.md

```diff
@@ -146,16 +148,16 @@ fn my_div(dividend: u32, divisor: u32) -> u32 {
    ```
 
    Kani verifies the expanded harness like any other harness, giving the
-   green light for the next step: verified stubbing.
+   green light for the next step: call abstraction.
 
 3. In the last phase the **verified** contract is ready for us to use to
-   **stub** other harnesses.
+   abstract the function at its call sites.
 
    Kani requires that there has to be at least one associated
-   `proof_for_contract` harness for each function to stub, otherwise an eror is
+   `proof_for_contract` harness for each abstracted function, otherwise an error is
    thrown. In addition, by default, it requires all `proof_for_contract`
    harnesses to pass verification before attempting verification of any
-   harnesses that use the contract as a stub.
+   harnesses that use the contract as an abstraction.
```

Comment:
I think stub is also more suitable here. The usages of abstraction above are fine.